helsingborg-stad/helsingborg-io-sls-api
# Feat/esbuild bundler

## Explain the changes you’ve made
Replaced serverless-bundle with serverless-esbuild as bundler

## Explain why these changes are made
Serverless-bundle can be very slow on some machines, making the deploy process a burden. Serverless-esbuild is proven to be a much faster bundler, hence changing to it instead.

## Explain your solution
Added the es-build configuration in the common serverless file in repository root, updated every stack serverless file to use esbuild instead of bundle.

## How to test

1. Checkout this branch
2. run `sls deploy` within service folder for `service x`
3. The service should be bundle:d much faster and the functionality should be the same without any errors

## Tested environments

- [x] Your personal AWS environment. -> Tested by deploying every stack and running the application without errors
- [] Your local Serverless environment.
